From 83cedaa4ab8c6cf7c45f9ffbbaed54aa8c0d3458 Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Mon, 23 Jan 2012 11:17:51 +0530
Subject: [PATCH 0742/1092] ARM: OMAP: id: Add missing break statement in
 omap3xxx_check_revision

Add missing break statement, which somehow got removed from
omap3xxx_check_revision function inside AM335x switch case,
resulting into wrong omap/cpu_rev configuration.
---
 arch/arm/mach-omap2/id.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-omap2/id.c b/arch/arm/mach-omap2/id.c
index 148bd1b..c66f700 100644
--- a/arch/arm/mach-omap2/id.c
+++ b/arch/arm/mach-omap2/id.c
@@ -398,6 +398,7 @@ void __init omap3xxx_check_revision(void)
 	case 0xb944:
 		omap_revision = AM335X_REV_ES1_0;
 		cpu_rev = "1.0";
+		break;
 	case 0xb8f2:
 		switch (rev) {
 		case 0:
-- 
1.7.11.2

